pkg/chaosdaemon/types.go

```diff
@@ -16,22 +16,34 @@
 package chaosdaemon
 
 import (
+	"time"
+
 	"github.com/vishvananda/netlink"
 
 	pb "github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb"
 )
 
 func ToNetlinkNetemAttrs(netem *pb.Netem) netlink.NetemQdiscAttrs {
```

Comment:
This method doesn't appear to be referenced anywhere within the Chaos Mesh codebase - all former references seem to have been refactored out.

Reply:
we could safely remove this file so :)

Reply:
Gone! Wasn't sure if it was retained for some sort of backwards compatibility, or just overlooked.